<?xml version="1.0" encoding="utf-8"?>
<launch>


  <node pkg="ros_aibot_drive" type="aibot_following.py" name="aibot_following" output="screen">
    <param name="Width" value="640" />
    <param name="Height" value="480" />
  </node>

</launch>
